conf/services.conf

```diff
@@ -0,0 +1,163 @@
+        restrictToTypes = [ photograph ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Creator = ^ca_objects.Creator,
+            DateOfCreation = ^ca_objects.DateOfCreation,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            Medium = ^ca_objects.Medium,
+            Subjects = ^ca_occurrences,
+        }
+    },
+    photographs_detail = {
+        type = detail,
+        table = ca_objects,
+        restrictToTypes = [ photograph ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Creator = ^ca_objects.Creator,
+            DateOfCreation = ^ca_objects.DateOfCreation,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            Medium = ^ca_objects.Medium,
+            PhysicalDescription = ^ca_objects.PhysicalDescription,
+            Subjects = ^ca_occurrences,
+            Summary = ^ca_objects.Summary,
+            Notes = ^ca_objects.Notes,
+            TermsOfUse = ^ca_objects.TermsOfUse
+        }
+    },
+
+    # Museum
+    museum_search = {
+        type = search,
+        table = ca_objects,
+        restrictToTypes = [ artwork, costume, museumArtefact ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            type = ^ca_objects.type_id,
+            ItemName = ^ca_objects.preferred_labels,
+            ItemDates = ^ca_objects.ItemDates,
+            Importance = ^ca_objects.Importance,
+            Subjects = ^ca_occurrences,
+            Classification = ^ca_objects.Classification
+        }
+    },
+    museum_detail = {
+        type = detail,
+        table = ca_objects,
+        restrictToTypes = [ artwork, costume, museumArtefact ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            type = ^ca_objects.type_id,
+            ItemName = ^ca_objects.preferred_labels,
+            ItemDates = ^ca_objects.ItemDates,
+            Description = ^ca_objects.Description,
+            MakersMarks = ^ca_objects.MakersMarks,
+            HistoricalDetails = ^ca_objects.HistoricalDetails,
+            BibliographReferences = ^ca_objects.BibliographReferences,
+            Importance = ^ca_objects.Importance,
+            Subjects = ^ca_occurrences,
+            StatementOfSignificance = ^ca_objects.StatementOfSignificance,
+            Classification = ^ca_objects.Classification
```

Comment:
There is no `Classification` attribute, this is done via a relationship to the controlled vocabulary. I _think_ this should be entered like:

`Classification = ^ca_list_items%restrictToRelationshipTypes=described`
but it might be:
```
Classification = <unit relativeTo="ca_list_items" restrictToRelationshipTypes="described">^ca_list_items.preferred_labels.name_singular</unit>
```